code/modules/mining/minebot.dm

```diff
@@ -279,10 +286,11 @@
 
 /obj/item/mine_bot_upgrade/proc/upgrade_bot(mob/living/simple_animal/hostile/mining_drone/M, mob/user)
 	if(M.melee_damage_upper != initial(M.melee_damage_upper))
-		to_chat(user, "<span class='warning'>[src] already has a combat upgrade installed!</span>")
+		to_chat(user, "<span class='warning'>The Minebot already has a combat upgrade installed!</span>")
```

Comment:
"The Minebot" could probably be changed here to the actual name of the minebot. More interaction with that rename board